apiserver/facades/client/sshclient/facade.go

```diff
@@ -379,7 +380,7 @@ func (facade *Facade) PublicHostKeyForTarget(arg params.SSHHostKeyRequestArg) pa
 		return res
 	}
 
-	res.HostKey = hostkey
+	res.KnownHostLine = knownHostLine
 
 	// Get controller jumpserver hostkey.
 	jumpHostKey, err := facade.backend.SSHServerHostKey()
```

Comment:
I went to go check whether this is a private/public key, and although the docstring for the interface says `SSHServerHostKey returns the public host key`. It is calling the state method `SSHServerHostKey()` which returns the private key of the jump server. So this needs to be fixed as well.

Additionally, this `sshclient` facade receives a model specific state, depending on which model the user was switched to. But the server private key is not in a model-specific state, it's in the system state as far as I understand. So I am curious if this would have even worked in the first place. Some QA steps that use postman or wscat to do a request/response would be helpful as we don't have cli side support for testing this yet.